transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java

```diff
@@ -296,6 +297,19 @@ private BatchedLogRequest getRequestBody(BackendRequest backendRequest) {
                   .setProductIdOrigin(ComplianceData.ProductIdOrigin.EVENT_OVERRIDE)
                   .build());
         }
+        if (eventInternal.getCookieOverride() != null) {
+          event.setZwiebackCookieOverride(eventInternal.getCookieOverride());
+        }
+
+        if (eventInternal.getExperimentIdsClear() != null
+            || eventInternal.getExperimentIdsEncrypted() != null) {
+          event.setExperimentIds(
```

Comment:
In this case both byte arrays are set if one is present, in later cases each one is checked and individually, is this fine?

Reply:
Fixed